community/contribution.md

```diff
@@ -15,28 +15,34 @@ _This document explains how Axelerant defines contributions with respect to KPIs
 
 An open source community contribution is defined as any piece of documentation (code or otherwise) or any offline activity that is documented and available in the public space for other community members to use.
 
+### Axelerant’s intent
+
+While we understand the above is an open source contribution **Axelerant’s intent is to encourage more meaningful contributions for the community & for the organisation.**
+
+## Important Notes before you continue reading:
```

Comment:
I'd suggest putting points 3 and 4 under "Axelerant's Intent" or a similar section after it. Point 5, if it really is a condition to getting the contribution accepted, should be moved under the KPI section. Points 1 and 2 can remain here and go to the top of the document or bottom.